(feat): add google colab env config (#225)
chalk-ai/docs: notebook-setup.mdx

import os

> from chalk import ChalkClient

Author: ```suggestion
from chalk.client import ChalkClient
```

The Chalk client can then be initialized referencing the variables directly

```python notebook.ipynb
from chalk.client import ChalkClient

client = ChalkClient(branch="new-feature")
```

### Vertex & Google Colab Enterprise

On Google Colab Enterprise, you can set up secrets using the Google Secret Manager.
Add secrets named `chalk-client-id` and `chalk-client-secret` to Google Secret Manager

> for your Chalk client id and client secret.

Reviewer: ```suggestion
that correspond to your Chalk client id and client secret.
```

![Google Colab Secret](/img/ColabSecret.png)

Add three dependencies to your notebook environment: `google-cloud-secret-manager`,
`google-cloud-resource-manager`, and `chalkpy`.

You can now read your credentials securely from the Google Secret Manager and
initialize your Chalk client!

```python
# !pip install google-cloud-secret-manager
# !pip install google-cloud-resource-manager
# !pip install chalkpy
import os

from google.cloud import secretmanager as sm
from google.cloud import resourcemanager_v3 as rm

from chalk.client import ChalkClient

# Create the Secret Manager client.
PROJECT_NAME = os.environ['GOOGLE_CLOUD_PROJECT']

# These should match the names given to your secrets in Google Secret Manager
client_id_name = "chalk-client-id"

> client_secret_name   = "chalk-client-secret"

Author: ```suggestion
client_secret_name  = "chalk-client-secret"
```

```python notebook.ipynb

> from chalk import ChalkClient

Reviewer 2: i thtitnk it's from chalk.client?

Author: Yup, nice catch

> from chalk import ChalkClient

Author: ```suggestion
from chalk.client import ChalkClient
```
